todo-app/src/app/todo.ts

```diff
@@ -0,0 +1,5 @@
+export interface Todo {
+  id?: number;
+  title?: string;
+  isDone?: boolean;
+}
```

Comment:
1. Rename Todo interface to TodoInterface
2. Create Class Todo with default values based on TodoInterface
3. Create a folder 'Models'. Rename todo.ts on todo.model.ts and move this file to "Models" folder

Reply:
Done.